imap_processing/tests/ena_maps/test_ena_maps.py

```diff
@@ -178,9 +178,16 @@ def test_project_pset_values_to_map_push_method(self, mock_frame_transform_az_el
         assert rectangular_map.data_dict != {}
 
         # Check that the map has the same values as the PSETs, summed
-        simple_summed_pset_counts = np.sum(
-            [pset["counts"].values for pset in self.l1c_pset_products], axis=0
-        ).reshape(rectangular_map.data_dict["counts"].shape)
+        simple_summed_pset_counts = np.zeros(rectangular_map.data_dict["counts"].shape)
```

Comment:
Do you want it to be the same datatype here too?
```suggestion
        simple_summed_pset_counts = np.zeros_like(rectangular_map.data_dict["counts"])
```

Reply:
I had been using `allclose`, so it wasn't explicitly necessary, but I can do this, and use `array_equal`!